schema/v1.1.0/common.yaml

```diff
@@ -740,6 +743,139 @@ slots:
     description: The mask label for a semantic segmentation mask annotation file.
     ifabsent: 'int(1)'
 
+  # ============================================================================
+  # Per Section Parameters fields
+  # ============================================================================
+  per_section_z_index:
+    range: int
+    description: z-index of the frame in the tiltseries
+    minimum_value: 0
+    required: true
+  per_section_defocus:
+    range: float
+    description: defocus amount
+    minimum_value: 100
+    maximum_value: 100
+    unit:
+      descriptive_name: microns
+      symbol: µm
+  per_section_astigmatism:
+    range: float
+    description: Astigmatism amount for this frame
+  per_section_astigmatic_angle:
+    range: float
+    description: Angle of ast
+    minimum_value: -180
+    maximum_value: 180
+    unit:
+      descriptive_name: degrees
+      symbol: °
+
+  # ============================================================================
+  # Alignment Parameters fields
+  # ============================================================================
+  alignment_alignment_type:
+    range: alignment_type_enum
+    description: Type of alignment included, i.e. is a non-rigid alignment included?
+
+  alignment_volume_x_dimension:
+    range: float
+    description: X dimension of the reconstruction volume in angstrom
+    unit:
+      descriptive_name: Angstrom
+      symbol: Å
+
+  alignment_volume_y_dimension:
+    range: float
+    description: Y dimension of the reconstruction volume in angstrom
+    unit:
+      descriptive_name: Angstrom
+      symbol: Å
+
+  alignment_volume_z_dimension:
+    range: float
+    description: Z dimension of the reconstruction volume in angstrom
+    unit:
+      descriptive_name: Angstrom
+      symbol: Å
+
+  alignment_volume_x_offset:
+    range: float
+    description: X shift of the reconstruction volume in angstrom
+    unit:
+      descriptive_name: Angstrom
+      symbol: Å
+
+  alignment_volume_y_offset:
+    range: float
+    description: Y shift of the reconstruction volume in angstrom
+    unit:
+      descriptive_name: Angstrom
+      symbol: Å
+
+  alignment_volume_z_offset:
+    range: float
+    description: Z shift of the reconstruction volume in angstrom
+    unit:
+      descriptive_name: Angstrom
+      symbol: Å
+
+  alignment_volume_x_rotation:
+    range: float
+    description: Additional X rotation of the reconstruction volume in degrees
+    unit:
+      descriptive_name: degrees
+      symbol: °
+
+  alignment_tilt_offset:
+    range: float
+    description: Additional tilt offset in degrees
+    unit:
+      descriptive_name: degrees
+      symbol: °
+
+  alignment_local_alignment_file:
```

Comment:
Honestly, I'm not sure if we even want this field at all - we decided to keep all of the alignment data directly in the db rather than relying on files in s3 for alignment data cc @uermel ?